dags/taar_daily.py

```diff
@@ -218,6 +218,70 @@
     dag=dag,
 )
 
+taar_lite_guidranking = SubDagOperator(
+    task_id="taar_lite_guidranking",
+    subdag=moz_dataproc_pyspark_runner(
+        parent_dag_name=dag.dag_id,
+        dag_name="taar_lite_guidranking",
+        default_args=default_args,
+        cluster_name=taarlite_cluster_name,
+        job_name="TAAR_Lite_GUID_Ranking",
+        python_driver_code="gs://moz-fx-data-prod-airflow-dataproc-artifacts/jobs/taar_lite_guidranking.py",
+        num_workers=8,
+        init_actions_uris=["gs://dataproc-initialization-actions/python/pip-install.sh"],
+        additional_metadata={'PIP_PACKAGES': "google-cloud-bigquery==1.20.0 google-cloud-storage==1.19.1 boto3==1.9.253"},
+        additional_properties={"spark:spark.jars.packages": "org.apache.spark:spark-avro_2.11:2.4.3",
+                               "spark:spark.jars":"gs://spark-lib/bigquery/spark-bigquery-latest.jar"},
+        master_machine_type='n1-standard-8',
+        worker_machine_type="n1-highmem-32",
+        py_args=[
+            "--date",
+            "{{ ds }}",
+            "--aws_access_key_id",
+            taar_aws_access_key,
+            "--aws_secret_access_key",
+            taar_aws_secret_key,
+        ],
+        aws_conn_id=taar_aws_conn_id,
+        gcp_conn_id=taar_gcpdataproc_conn_id,
+        master_disk_type="pd-ssd",
```

Comment:
maybe comment how this is to make the job run faster

Reply:
i added comments in now